app.yaml

```diff
@@ -0,0 +1,32 @@
+runtime: nodejs12
+
+handlers:
+  - url: /static/js/(.*)
+    static_files: build/static/js/\1
+    upload: build/static/js/(.*)
+    secure: always
+    redirect_http_response_code: 301
+  - url: /images/(.*\.(gif|png|jpg))$
+    static_files: public/images/\1
+    upload: public/images/.*\.(gif|png|jpg)$
+    secure: always
+    redirect_http_response_code: 301
+  - url: /(.*\.(json|ico|png|txt))$
+    static_files: build/\1
+    upload: build/.*\.(json|ico|png|txt)$
+    secure: always
+    redirect_http_response_code: 301
+  - url: /api(\/.*)*
+    script: auto
+    secure: always
+    redirect_http_response_code: 301
+  - url: /.*
```

Comment:
This file is configured to serve react build as static files in every path, except for `/api`, in which we will expose the api endpoints.

You can try the frontend [here](https://enamel-step-2020.appspot.com/) or at any subpath, like [this](https://enamel-step-2020.appspot.com/example).

And finally, the api is working [here](https://enamel-step-2020.appspot.com/api)